config/twig_hooks/admin/product/create.yaml

```
@@ -0,0 +1,6 @@
+sylius_twig_hooks:
+    hooks:
+        'sylius_admin.product.create.content.form.sections.general':
+            commodity_code:
+                template: '@SyliusAdyenPlugin/product/form/sections/general/commodity_code.html.twig'
```

```suggestion
                template: '@SyliusAdyenPlugin/admin/product/form/sections/general/commodity_code.html.twig'
```
and the same in other configs